### ISSUE TITLE
Social buttons

### ISSUE THREAD

**Issue by abhranildas** [dvidby0/artarium#71]
For each artist page definitely.
For each photo page?
For artarium homepage?

**Comment by xcitech** [comment]
"For each artist page definitely."
Does this direct to the artist's personal facebook and twitter page?

**Comment by abhranildas** [comment]
No. Here I'm talking about 'like' buttons and such. Clicking the like/share button on the artist page likes/shares that artist page.
